lib/vfs/src/host_fs.rs

```diff
@@ -9,6 +9,8 @@ use std::fs;
 use std::io::{self, Read, Seek, Write};
 #[cfg(unix)]
 use std::os::unix::io::{AsRawFd, RawFd};
+#[cfg(target_arch = "wasm32")]
```

Comment:
These vfs changes shouldn't be needed any more now that wasmer-compiler-cli doesn't depend on vfs.

Reply:
ok, done.